### ISSUE TITLE
Still 'extension is not provided' output for most probably available extensions

### ISSUE THREAD

**Issue by lbayerl** [cloudfoundry/php-buildpack#196]
What version of Cloud Foundry and CF CLI are you using?
Api Version: 2.65.0
cf version 6.25.0+787326d.2017-02-28

What version of the buildpack you are using?
4.3.28

If you were attempting to accomplish a task, what was it you were attempting to do?
pushing a PHP app

What did you expect to happen?
according to the release notes of the [release notes](https://github.com/cloudfoundry/php-buildpack/releases/tag/v4.3.28) php buildpack 4.3.28 should not complain anymore about modules which are compiled with PHP already during push ([Ticket](https://www.pivotaltracker.com/n/projects/1042066/stories/138757115))

What was the actual behavior?
during push I saw the following statement:
```bash
Downloaded [https://buildpacks.cloudfoundry.org/dependencies/php7/php7-7.0.16-linux-x64-9ac5055c.tgz] to [/tmp]
The extension 'spl' is not provided by this buildpack.
The extension 'simplexml' is not provided by this buildpack.
```
I am wondering whether spl and simplexml aren't really compiled with the php buildpack because:
'spl': This extension is available and compiled by default in PHP 5.0.0 and you can't even deactivate it. [Manual](http://php.net/manual/de/spl.installation.php)
'simplexml': This extension is enabled by default. This buildpack would need to disable it intentionally. [Manual](http://php.net/manual/en/simplexml.installation.php)

So is it possible that  ([Ticket](https://www.pivotaltracker.com/n/projects/1042066/stories/138757115) doesn't fix all of the required output?

Please confirm where necessary:
* [X] I have included a log output
* [ ] My log includes an error message
* [ ] I have included steps for reproduction

**Comment by sclevine** [comment]
Thanks for the report! We'll investigate.

**Comment by sclevine** [comment]
We determined that this was caused by a bug that affected detecting extensions with uppercase characters. Fixed in the latest release: [v4.3.29](https://github.com/cloudfoundry/php-buildpack/releases/tag/v4.3.29)
